RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -521,26 +536,37 @@ private void DisplayTreeSearchBox()
 
             GUILayout.BeginHorizontal();
             {
-                if (GUILayout.Button("Search scene"))
+                GUILayout.Label("Search");
```

Comment:
Added a label here to remove the word search from each button to make room and then labelled the buttons Name/Component/Properties/Static. Fits nicely.